data/json/professions.json

```diff
@@ -4147,5 +4147,21 @@
       "female": [ "sports_bra", "boy_shorts" ]
     },
     "flags": [ "SCEN_ONLY" ]
+  },
+  {
+    "type": "profession",
+    "ident": "knight",
+    "name": "Medieval Knight",
+    "description": "You've made medieval swordsmanship your life. Now you will finally have a real use for it.",
```

Comment:
```suggestion
    "description": "You've made medieval swordsmanship your life.  Now you will finally have a real use for it.",
```

Reply:
I'm sorry if I'm misunderstanding something here but why does it need an extra space there?

Reply:
It's a style standard we have - two spaces after a period. It has something to do with monospaced fonts.

Reply:
Alright, thanks.